### ISSUE TITLE
Narrator does not announce loading status for Picker component

### ISSUE THREAD

**Issue by Zeddox** [microsoft/fluentui#21912]
### Environment Information

- **Package version(s)**: @fluentui/react 8.56.0
- **Browser and OS versions**:  Edge - Windows 10

### Describe the issue:

Accessibility testing within our application has surfaced that in our Picker component usage, when suggestions are being loaded after someone has entered a search term, that the loading status is not announced by Narrator.

Accessibility test team provided this link for WCAG: https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

Example: https://codepen.io/zeddox/pen/XWzyBXO

#### Actual behavior:

Typing into the TagPicker opens the suggestions callout and the Narrator announces: Suggestions Available

#### Expected behavior:

Typing into the TagPicker opens the suggestions callout and the Narrator announces the status: Loading Items

**Comment by micahgodbolt** [comment]
@Zeddox i've got a PR out for this issue. Tell me if you'd expect the `loadingText` to always match the `ariaText`, or if you want an explicit aria prop. 

As for the "Suggestions Available", that seems to be a baked in response to the 'aria-controls' related div popping up. I don't see that string anywhere. Still a little baffled by that, but with this fix it will read out the `loadingText`

**Comment by Zeddox** [comment]
@micahgodbolt Thanks for hopping on this. We don't need any different aria text prop in our case, `loadingText` being passed works for now.

I suppose if one were to have `loadingText: 'Loading...'` that would sound odd announced and may warrant an aria prop.
